app/services/permitted_attributes/business_address.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module PermittedAttributes
+    class BusinessAddress
+      def self.attributes
+        [
+          :business_name, :address1, :address2,
+          :city, :country_id, :state_id, :zipcode,
+          :phone
+        ]
+      end
+    end
+  end
+  
```

Comment:
You're missing an `end` here :eyes:

Reply:
You got a stray space in the last line of the file.